examples/src/main/python/worker_memory_check.py

```diff
@@ -0,0 +1,47 @@
+#
```

Comment:
Is examples the right place for this?

Reply:
Easiest place to put to be trivially picked up by integration tests, as I did with pyfiles, but I am open to recommendations.

Reply:
shouldn't this be in python tests (and get it to run only certain cluster manager)

Reply:
That might be a good place for it. But the reason for this being in examples is that the integration tests can access this locally and we can see the success in the Jenkins environment. The K8s integration test suite does not run python run-tests.py which means that this test would therefore not be part of the PRB.